quickstart/test/compose-test.zsh

```diff
@@ -68,35 +68,37 @@ if [[ -n "${ZITI_QUICK_DIR:-}" ]]; then
     for FILE in "${QUICK_FILES[@]}"; do
         cp "${ZITI_QUICK_DIR}/${FILE}" .
     done
-    if [[ -n "${ZITI_QUICK_IMAGE_TAG:-}" ]]; then
+    if [[ -n "${ZITI_QUICK_TAG:-}" ]]; then
         if [[ -x "${ZITI_QUICK_DIR:-}/docker/createLocalImage.sh" ]]; then
             (
                 cd "${ZITI_QUICK_DIR}/docker"
                 unset ZITI_VERSION ZITI_OVERRIDE_VERSION  # always build the local source
-                ./createLocalImage.sh --build "${ZITI_QUICK_IMAGE_TAG}"
+                ./createLocalImage.sh --build "${ZITI_QUICK_TAG}"
             )
         else
-            echo "ERROR: ZITI_QUICK_IMAGE_TAG is set but ZITI_QUICK_DIR/docker/createLocalImage.sh is not executable" >&2
+            echo "ERROR: ZITI_QUICK_TAG is set but ZITI_QUICK_DIR/docker/createLocalImage.sh is not executable" >&2
             exit 1
         fi
     fi
-elif [[ -n "${ZITI_QUICK_IMAGE_TAG:-}" ]]; then
-    echo "ERROR: ZITI_QUICK_IMAGE_TAG is set but ZITI_QUICK_DIR is not set" >&2
+elif [[ -n "${ZITI_QUICK_TAG:-}" ]]; then
+    echo "ERROR: ZITI_QUICK_TAG is set but ZITI_QUICK_DIR is not set" >&2
     exit 1
 else
-    echo "ERROR: ZITI_QUICK_IMAGE_TAG is not set, try running with --local" >&2
+    echo "ERROR: ZITI_QUICK_TAG is not set, try running with --local" >&2
     exit 1
 fi
 
 # rename the simplified Compose file to the default Compose project file name
 mv ./simplified-docker-compose.yml ./compose.yml
 
 # learn the expected Go version from the Go mod file so we can pull the correct container image
-ZITI_GO_VERSION="$(awk '/^go[[:space:]]+/ {print $2}' ./go.mod)"
+: ${ZITI_GO_VERSION:="$(awk '/^go[[:space:]]+/ {print $2}' ./go.mod)"}
```

Comment:
use parent var if defined

Reply:
> use parent var if defined

i'd rather see an echo statement here regardless of how it was found. ideally it should tell you if it's getting it from the environment or if it determining it by reading the go.mod on its own